src/gpo/assets/PowerToys.admx

```diff
@@ -240,6 +240,16 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+    <policy name="ConfigureEnabledUtilityPeek" class="Both" displayName="$(string.ConfigureEnabledUtilityPeek)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityPeek">
+      <parentCategory ref="PowerToys" />
+      <supportedOn ref="SUPPORTED_POWERTOYS_0_64_0" />
```

Comment:
We need `0_70_0`. Pleas add a `SupportedOn` definition and the string in the adml too.

Reply:
Thank you for the comment!
I didn't clearly catch the purpose of changing the comment on line 8, should it be changed every time or just when the minRequiredRevision is changed?

Reply:
The comment on line 8 is for the end user/admin to know when the GPOs are changed/updated the last time. (Maybe it makes more sense to align the revision to PTs version number.  But this is a different thing.)

Reply:
Thanks for clarifying :)

Reply:
Definitely in the future it might be worthwhile to align the revision to PTs version number.
0.69 got released with revision 1.1 and it should've been updated

Reply:
In the past we didn't update this number.